spec/support/capybara.rb

```diff
@@ -0,0 +1,5 @@
+
+require_relative 'locators/find_table_row'
+
+Capybara::Session.include FindTableRow
```

Comment:
can we remove this line ?

Reply:
we checked this but we don't have access to this method without this line. In feature spec method have_table_row does not work